node/lib/util/submodule_util.js

```diff
@@ -559,11 +559,19 @@ exports.getSubmodulesInPath = function (dir, indexSubNames) {
     if ("" === dir) {
         return indexSubNames;
     }
+
+    // test if the short path a parent dir of the long path
+    const isParentDir = (short, long) => {
+        return long.startsWith(short) && (
+            short[short.length-1] === "/" ||
+            long.replace(short, "")[0] === "/"
```

Comment:
this check could be long[short.length] === '/', right?

Reply:
yah, logically they are the same, but yours is conciser!